### PR TITLE
MONIT-26972: Related links are not grouped.

### PR DESCRIPTION
**Pull request by karenghazaryan**

### CODE REVIEW

File: onboarding/welcome.md [hunk 1]
```diff
@@ -13,9 +13,9 @@
 &nbsp;
 &nbsp;
 
-<p>Have questions? &nbsp; &nbsp; <a href="https://wavefront.com/join-public-slack"> <img src="images/slack.png"></img></a>
+<p>Have questions? &nbsp; &nbsp; <a href="https://wavefront.com/join-public-slack" aria-label="Slack us"> <img src="images/slack.png" alt="Slack icon"></img></a>
 &nbsp; &nbsp;
-<a href="https://wavefront.com/meet-1-on-1"><img src="images/1on1.png"></img> </a> </p>
+<a href="https://wavefront.com/meet-1-on-1" aria-label="Schedule one on one"><img src="images/1on1.png"></img> </a> </p>
```
Comment (mmihaylovam) [inline onboarding/welcome.md:18]: Do we need an alt text for the image?

File: onboarding_v2/next_steps.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 <p>&nbsp;</p>
 <p><strong>Invite your co-workers to collaborate in Wavefront!</strong></p>
 <p>Enter email addresses separated by commas or spaces.</p>
 
 [[inviteUsers]]
 
 <p>Want to learn about other features? Have questions?</p>
-<p><a href="https://docs.wavefront.com/tutorial_overview.html"><img src="images/tut_snag.png"></img></a>&nbsp; &nbsp;
-<a href="https://help.wavefront.com/hc/en-us/requests/new"> <img src="images/get_help_snag.png"></img></a>&nbsp; &nbsp;
+<p><a href="https://docs.wavefront.com/tutorial_overview.html" aria-label="Tutorial and videos"><img src="images/tut_snag.png" alt="Tutorial and videos"></img></a>&nbsp; &nbsp;
```
Comment (mmihaylovam) [inline onboarding_v2/next_steps.md:12]: `aria-label="Tutorial and videos"` -> `aria-label="Tutorials and videos"`
`alt="Tutorial and videos"` -> `alt="Video icon"` (and consistency with onboarding_v2/welcome.md)

File: onboarding_v2/next_steps.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+<a href="https://help.wavefront.com/hc/en-us/requests/new" aria-label="Get help"> <img src="images/get_help_snag.png" alt="Get help"></img></a>&nbsp; &nbsp;
```
Comment (mmihaylovam) [inline onboarding_v2/next_steps.md:13]: `alt="Get help"` -> `alt="Help icon"` (For consistency with onboarding_v2/welcome.md?)

File: onboarding_v2/welcome.md [hunk 4]
```diff
@@ -9,7 +9,7 @@
 </ul>
 <p>&nbsp;</p>
 <p>Have questions?&nbsp; &nbsp;
-<a href="https://docs.wavefront.com/tutorial_overview.html"><img src="images/tut_snag.png"></img></a> &nbsp; &nbsp;<a href="https://help.wavefront.com/hc/en-us/requests/new"> <img src="images/get_help_snag.png"></img></a></p>
+<a href="https://docs.wavefront.com/tutorial_overview.html" aria-label="Tutorial and videos"><img src="images/tut_snag.png" alt="Tutorial icon"></img></a> &nbsp; &nbsp;<a href="https://help.wavefront.com/hc/en-us/requests/new" aria-label="Get help"> <img src="images/get_help_snag.png" alt="Help icon"></img></a></p>
```
Comment (mmihaylovam) [inline onboarding_v2/welcome.md:12]: `aria-label="Tutorial and videos"` -> `aria-label="Tutorials and videos"`
`alt="Tutorial icon"` -> `alt="Video icon"`

File: onboarding_v2/next_steps.md (hunk 3 above)
Comment (karenghazaryan) [reply onboarding_v2/next_steps.md:13]: These two variants are possible. I changed them for consistency

**Review by mmihaylovam — APPROVED** [review]

**Pull request merged**